Dangerfile

```diff
@@ -0,0 +1 @@
+code_style_validation.check file_extensions: ['.cc', '.cpp', '.m', 'mm', '.h', '.java']
```

Comment:
Why adding this file?

Reply:
because code format validator is based on danger plugin: code_style_validation
and this line in Dangerfile is to start format check.

Reply:
I am not that familiar with danger, please let me know when this `Dangerfile` will be executed.

Reply:
here is the format code part in .travis I list above:

```
matrix:
    fast_finish: true
    include:
      # format code using clang-format
      - env: TEST_SUITE=code_format
        language: ruby

install:
    elif[[ ("$TEST_SUITE" = "code_format") ]]; then
        bundle install
    fi

script:
    if [[ ("$TEST_SUITE" = "code_format") ]]; then
        echo "exec danger ruby"
        bundle exec danger
    fi
```

Reply:
yes, generally no one will read the raw log of Travis if build is success, but this also means that oclint only output warning message.

but the oclint logs is over 1000 lines ,if we put them in danger, is too long !

maybe we can update the input file of oclint from the whole `ios-sdk` project to the file that changed of pr.or we can change the oclint rules to 4 rules may lead to bug or crash.in this case, the log will be short enougn to output by danger.

Reply:
yes, it rely on https://github.com/wrmswindmill/danger-code_style_validation.git, which I fork from https://github.com/flix-tech/danger-code_style_validation and I update some code for weex project. I will share this plugin in the mailing list  as well as the static check in Travis CI.

here are some changes I make for danger-code_style_validation:
1. change message level from error to warn
2. modify the message hint:
```
If you have not install clang-format, you can install clang-format by npm install -g clang-format
Then Execute one of the following actions and commit again:

Run clang-format -i on the offending files
Apply the suggested patches with git apply patch.
If you use Xcode,you can also use clang-format plugin: https://github.com/travisjeffery/ClangFormat-Xcode
```

the below is my answer for your question：
1. there are not output of the code format in danger because every file I changed is  conform with the regulations ，here is the demo of it：https://github.com/wrmswindmill/incubator-weex/pull/13#issuecomment-516279338
2. I provide the extra message. Xcode has plugin but there are no plugin for android studio.